Makefile

```diff
@@ -151,12 +151,8 @@ $(IMAGES_PACKAGE_DIR): | $(LOCALBIN)
 
 TEMPLATE_FOLDERS = $(patsubst $(TEMPLATES_DIR)/%,%,$(wildcard $(TEMPLATES_DIR)/*))
 
-.PHONY: collect-airgap-providers
-collect-airgap-providers: yq helm clusterctl $(PROVIDER_TEMPLATES_DIR) $(LOCALBIN)
-	$(SHELL) hack/collect-airgap-providers.sh
-
 .PHONY: helm-package
-helm-package: $(CHARTS_PACKAGE_DIR) $(EXTENSION_CHARTS_PACKAGE_DIR) helm collect-airgap-providers
+helm-package: $(CHARTS_PACKAGE_DIR) $(EXTENSION_CHARTS_PACKAGE_DIR) helm
 	@make $(patsubst %,package-%-tmpl,$(TEMPLATE_FOLDERS))
 
 .PHONY: k0s-image
```

Comment:
This target related to airgap. Should be removed
Related variables (`K0S_AG_IMAGE` and `K0S_VERSION`)must be removed as well 
https://github.com/k0rdent/kcm/pull/938/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R140-R141